packages/@okta/vuepress-site/docs/api/resources/oidc/index.md

```diff
@@ -1338,33 +1208,69 @@ For more information about configuring an app for OpenID Connect, including grou
-using either of the following methods:
+* `none` - Use this method when the client doesn't authenticate itself to the `/token` endpoint because it uses the [Implicit Flow](/authentication-guide/implementing-authentication/implicit/) or because it is a public client with no client secret or other authentication mechanism.
 
-* Provide `client_id` and `client_secret`
-  in an Authorization header in the Basic auth scheme (`client_secret_basic`). For authentication with Basic auth, an HTTP header with the following format must be provided with the POST request:
+> Note: You can use only one of these methods in a single request or an error occurs.
+
+### Client Secret
+If you configured your client to use a `client_secret` [client authentication method](http://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication), provide the `client_id` and `client_secret` using one of these methods: 
+
+* `client_secret_basic`: Provide the `client_id` and `client_secret` values in the Authorization header as a Basic auth base64-encoded string with the POST request:
   ```bash
-  Authorization: Basic ${Base64(<client_id>:<client_secret>)}
+    Authorization: Basic ${Base64(<client_id>:<client_secret>)}
   ```
-* Provide the `client_id` and `client_secret`
-  as additional parameters to the POST body (`client_secret_post`)
-* Provide `client_id` in a JWT that you sign with the `client_secret`
-  using HMAC algorithms HS256, HS384, or HS512. Specify the JWT in `client_assertion` and the type, `urn:ietf:params:oauth:client-assertion-type:jwt-bearer`, in `client_assertion_type` in the request.
+* `client_secret_post`: Provide the `client_id` and `client_secret` as additional parameters in the POST request body.
 
-If you configure your client to use `private_key_jwt` [Client Authentication Method](http://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication) you must provide the `client_id` in a JWT that you sign with your private key.
-* The private key that was used to sign must have the corresponding public key registered in the client's [JWKSet](/docs/api/resources/oauth-clients/#json-web-key-set).
-* Sign the JWT with one of the following signature algorithms: RS256, RS384, RS512, ES256, ES384 or ES512.
-* Specify the JWT in `client_assertion` and the type, `urn:ietf:params:oauth:client-assertion-type:jwt-bearer`, in `client_assertion_type` in the request.
+* `client_secret_jwt`: Provide the `client_id` in a JWT that you sign with the `client_secret` using an HMAC SHA algorithm (HS256, HS384, or HS512). The JWT also contains other values, such as issuer and subject. See [Token Claims for Client Authentication with Client Secret or Private Key JWT](/docs/api/resources/oidc/#token-claims-for-client-authentication-with-client-secret-or-private-key-jwt). 
```

Comment:
Let's split this into another h3: JWT with Shared Key

We could rename the following h3 to JWT with Private Key for consistency.

Reply:
DONE